### PR TITLE
Add OPDS 2.0 import support

### PR DESCRIPTION
**Pull request by vbessonov**
## Description

This PR adds OPDS 2.0 support to Circulation Manager:
- adds `OPDS2Importer` implemented in [PR 1205](https://github.com/NYPL-Simplified/server_core/pull/1205) to a list of available collection types
- adds `opds2_import_monitor` script

## Checklist:

- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by leonardr — APPROVED** [review]
This looks fine to merge once core is merged.

**Comment by leonardr** [comment]
I think you need to update the submodule in this branch.

**Comment by vbessonov** [comment]
@leonardr, I updated the submodule

**Pull request merged**
